script.js

```diff
@@ -54,4 +58,33 @@ const menuDisplay = () => {
 
   deleteBtn.addEventListener('click', removeModal);
 };
-menuBtn.addEventListener('click', menuDisplay);
\ No newline at end of file
+menuBtn.addEventListener('click', menuDisplay);
+
+// ---------------form validation
+
+// const email = document.querySelector('#e-mail');
+// const emailMob = document.querySelector('#e-mail-mob');
+// const messageMob = document.querySelector('.message-mob');
+// const messageDesk = document.querySelector('.message-desk');
+// const formDesktop = document.querySelector('.main-form');
+// const formMobile = document.querySelector('.form-mobile');
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

Reply:
resolved and thanks